Cargo.toml

```diff
@@ -11,23 +11,23 @@ name = "rnix-lsp"
 version = "0.3.0-dev"
 
 [dependencies]
-dirs = "3.0.2"
+dirs = "4.0.0"
 env_logger = "0.9.0"
-gc = { version = "0.4", features = ["derive"] }
-lazy_static = "1.4"
-libc = "0.2.66"
-log = "0.4.8"
-lsp-server = "0.5.2"
-lsp-types = { version = "0.89.2", features = ["proposed"] }
-regex = "1"
+gc = { version = "0.4.1", features = ["derive"] }
+lazy_static = "1.4.0"
+libc = "0.2.126"
+log = "0.4.17"
+lsp-server = "0.6.0"
+lsp-types = { version = "0.93.0", features = ["proposed"] }
+regex = "1.5.6"
 rnix = "0.10.2"
-rowan = "0.12.6"
```

Comment:
Is `rowan` even used here? If yes, I'd be surprised that this works fine considering that `rnix-parser` is still using 0.12.

Reply:
According to github search its not used here and was added in https://github.com/nix-community/rnix-lsp/commit/bcf8963ce865c4f94c7e01e63861b87e48b71188

Reply:
OK, I'd be in favor of removing it then. Rest LGTM.